### DISCUSSION TITLE
Angular 13 with WebPack 5, cannot fix polyfill error

### DISCUSSION THREAD

**Discussion by fablb** [ghiscoding/Angular-Slickgrid#1178] [Q&A] [answered]
Hi,

I'm trying to install Angular-Slickgrid v4.3.1 using Angular v13.3.11.
I encoutered the polyfill error mentionned in the [HOWTO wiki](https://github.com/ghiscoding/angular-slickgrid/wiki/HOWTO---Step-by-Step) and followed the steps to fix it nevertheless the error still shows up.

Here's the error in detail:
```
./node_modules/aws4/aws4.js:4:13-30 - Error: Module not found: Error: Can't resolve 'crypto' in '[...]\node_modules\aws4'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "crypto": require.resolve("crypto-browserify") }'
        - install 'crypto-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "crypto": false }
```

**Comment by ghiscoding** [comment, marked as answer]
Well I don't think you followed all the steps, this error is from WebPack 5 itself (not from Angular-Slickgrid) and the steps to fix it is shown in the HOWTO at step 12
https://github.com/ghiscoding/angular-slickgrid/wiki/HOWTO---Step-by-Step#12-angular-12-with-webpack-5---how-to-fix-polyfill-error

it's also mentioned in main readme page
https://github.com/ghiscoding/Angular-Slickgrid#angular-12-with-webpack-5---how-to-fix-polyfill-error

and is also mentioned in the Version Compatibility Wiki page
https://github.com/ghiscoding/angular-slickgrid/wiki/Versions-Compatibility-Table#angular-12-with-webpack-5---how-to-fix-polyfill-error

Follow these instructions and if that doesn't work then search on the internet because like I said this is a WebPack 5 breaking change error and is unrelated to Angualr-Slickgrid. It's basically polyfills that were removed in WebPack 5 and must be installed manually now. The instructions I have is for `stream` but it should be the same for other missing polyfill like `crypto`
